pkgs/development/octave-modules/fem-fenics/default.nix

```diff
@@ -0,0 +1,32 @@
+{ buildOctavePackage
+, stdenv
+, fetchurl
+, dolfin
+, fenics
+, pkg-config
+}:
+
+buildOctavePackage rec {
+  pname = "fem-fenics";
+  version = "0.0.5";
+
+  src = fetchurl {
+    url = "https://octave.sourceforge.io/download.php?package=${pname}-${version}.tar.gz";
+    sha256 = "1xd80nnkschldvrqx0wvrg3fzbf8sck8bvq24phr5x49xs7b8x78";
+  };
+
+  buildInputs = [
+    dolfin
+    fenics
+    pkg-config
```

Comment:
Probably nativeBuildInputs.

Reply:
Yes. I just need to have fenics as a library that I can access. Right now it's only present if `pythonPackages.fenics` is present, but I have no way to access `fenics`, so I will leave the `broken = true` label for now.